stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt

```diff
@@ -75,7 +77,7 @@ public data class StreamDimens
     "This constructor has been deprecated. Parameter 'attachmentsContentImageHeight' has been deprecated in" +
         " favor of 'attachmentsContentImageMaxHeight'. Please use the constructor which does not contain " +
         "`attachmentsContentImageHeight`",
-    level = DeprecationLevel.WARNING
+    level = DeprecationLevel.ERROR,
```

Comment:
@MarinTolic We deprecated the imageHeight parameter for Compose and because of this the constructor.

Should we then, in the `default()` dimens implementation, choose to use the new constructor? As the new one is unused currently and the value we deprecated is still there.

Reply:
Good point.

@adasiewiczr When you see this please let me know if you want me to update the PR.